### DISCUSSION TITLE
Auto submit desync debug logs to the developers

### DISCUSSION THREAD

**Discussion by ageekhere** [OpenRCT2/OpenRCT2#16694] [Feature requests]
When a player gets a desync are the debug logs sent to the developers or do users have to manually submit desync logs via Github?  If not maybe a option could be added to the desync pop up box to submit desync logs to the developers so they can better find and fix desyncs.

**Comment by ZehMatt** [comment]
There is an option to debug desyncs but they have to be active on the server and are disabled by default for performance reasons. If the server can handle it just fine then I would encourage the owners to enable it in the config.ini

Its called
```desync_debugging``` and when its set to true the server keeps a small amount of older simulation states around that the client can ask for during a desync to see which data mismatches. Doing this is unfortunately quite expensive so its more a debugging tool if we know how to reproduce the desync.
